alerta/database/backends/postgres/base.py

```diff
@@ -1659,3 +1661,46 @@ def _deleteall(self, query, vars, returning=False):
     def _log(self, cursor, query, vars):
         current_app.logger.debug('{stars}\n{query}\n{stars}'.format(
             stars='*' * 40, query=cursor.mogrify(query, vars).decode('utf-8')))
+
+    def _apply_migration(self, conn, migration):
+        ver, name = migration.stem.split('_', 1)
+
+        cursor = conn.cursor()
+        cursor.execute("SELECT 1 FROM migrations WHERE version=%s", (ver,))
+        if cursor.fetchone():
+            return
+
+        try:
+            with open(migration.absolute()) as f:
+                cursor.execute(f.read())
+                cursor.execute("INSERT INTO migrations (version, name) VALUES (%s, %s)", (ver, name))
+                conn.commit()
+        except Exception as e:
+            conn.rollback()
+            current_app.logger.error('Failed to apply migration %s: %s', migration, e)
```

Comment:
Preferably remove logging from here and leave the info in the exception so the `create_engine` method can handle any errors